website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -20,6 +20,8 @@ Follow the steps below to configure Appsmith to use an external MongoDB instance
 - A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) for installing Appsmith.
 - Ensure that your external MongoDB has a replica set configuration in place. Connect to your database as an admin user and run [rs.initiate()](https://docs.mongodb.com/manual/reference/method/rs.initiate/). Note that if you're using MongoDB Cloud, the replica set configuration is already set up for you.
 - Ensure the MongoDB user account has `readWrite` and `clusterMonitor` roles assigned.
+- At least 2 GB of free storage space for backup and update tasks.
+- Create a backup of the Appsmith instance, See [Backup instance](/getting-started/setup/instance-management/appsmithctl#backup-instance) guide. (Recommended)
```

Comment:
@sharat87 - Added the step for backup.

Reply:
I presume we have to add steps for restoring the data from embedded  MongoDB to External. Could you please share those steps.

Reply:
We have a doc on both backup and restore, right? Can we instead mention something like this?

> Before you switch your MongoDB, please use the Backup and Restore guide to copy your data from your old MongoDB to the new one.

Reply:
Yes, we can. So they don't need to take the backup of instance - which we are doing for all the other docs. In this case, we will just link them to - [Backup database](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-database) and[ Restore database](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#restore-database)

Reply:
Reworded. Review.